testsuite/features/secondary/srv_manage_activation_keys.feature

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2010-2019 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Be able to manipulate activation keys
```

Comment:
Feature: Use activation keys

Reply:
Actually @Bischoff , I prefer "manipulate" as it reflects better what we do in those scenarios.